packages/lesswrong/server/utils/feedUtil.ts

```diff
@@ -49,6 +52,42 @@ export function viewBasedSubquery<
   };
 }
 
+export function fieldChangesSubquery<N extends CollectionNameString>({type, collection, context, documentIds, fieldNames}: {
+  type: string,
+  collection: CollectionBase<N>,
```

Comment:
Do we even use the collection for anything inside of the function?  If it's only used to constrain `fieldNames`, maybe better to have it accept `collectionName`, which is lighter-weight (and doesn't require importing additional collections upstream)